recipes/recipes_emscripten/qutip/recipe.yaml

```diff
@@ -0,0 +1,49 @@
+context:
+  version: "4.7.0"
+
+package:
+  name: qutip
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/qutip/qutip/archive/v{{ version }}.tar.gz
+  sha256: b942c9767281c78be92bea8a0db4e0a97af1d81aa67128601d3272b22c25d848
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"  # [not win]
+
+requirements:
+  build:
+    - sel(build_platform != target_platform): python
+    - sel(build_platform != target_platform): cross-python_{{ target_platform }}
+    - sel(build_platform != target_platform): cython
+    - sel(build_platform != target_platform): numpy
```

Comment:
Ooh. What do these `sel(...)` things do?

Reply:
That is just like the `# [...]` selector comments of the "meta.yaml" format. However, this syntax parses as proper YAML.